www/src/templates/installation.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+*/
+
+'use strict';
+
+import Container from 'components/Container';
+import Flex from 'components/Flex';
+import MarkdownHeader from 'components/MarkdownHeader';
+import NavigationFooter from 'templates/components/NavigationFooter';
+import {StickyContainer} from 'react-sticky';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import StickySidebar from 'components/StickySidebar';
+import findSectionForPath from 'utils/findSectionForPath';
+import {sharedStyles} from 'theme';
+
+import sectionListA from '../../../docs/_data/nav_docs.yml';
+import sectionListB from '../../../docs/_data/nav_contributing.yml';
+
+const sectionList = sectionListA
+  .map(item => {
+    item.directory = 'docs';
+    return item;
+  })
+  .concat(
+    sectionListB.map(item => {
+      item.directory = 'contributing';
+      return item;
+    }),
+  );
+
+// HACK: copied from 'installation.md'
+// TODO: clean this up.
+function setSelected(value) {
+  var tabs = document.querySelectorAll('li[role="tab"]');
+  for (var i = 0; i < tabs.length; ++i) {
+    var tab = tabs[i];
+    if (tab.className === 'button-' + value) {
+      tabs[i].setAttribute('aria-selected', 'true');
+      tabs[i].setAttribute('tabindex', '0');
+    } else {
+      tabs[i].setAttribute('aria-selected', 'false');
+      tabs[i].setAttribute('tabindex', '-1');
+    }
+  }
+}
+
+function keyToggle(e, value, prevTab, nextTab) {
+  // left arrow <-
+  if (e.keyCode === 37) {
+    document.getElementById(prevTab).focus();
+    display('target', prevTab);
+  }
+  // right arrow ->
+  if (e.keyCode === 39) {
+    document.getElementById(nextTab).focus();
+    display('target', nextTab);
+  }
+}
+window.keyToggle = keyToggle;
```

Comment:
Whoops! I missed this.

We can't have `window` references in code that's auto-executed as part of an import b'c they break the Gatsby `build` step. I'll remove this one! 😄